app/src/interfaces/select-dropdown-m2o/select-dropdown-m2o.vue

```diff
@@ -186,6 +186,8 @@ const edits = computed(() => {
 
 function onPreviewClick() {
 	if (props.disabled) return;
+	// prevent double dialog in case the edit dialog is already open
```

Comment:
To follow the general comment style
```suggestion
	// Prevent double dialog in case the edit dialog is already open
```

Reply:
I dont see why, but i also dont see why not 😂